### ISSUE TITLE
Missing log message when using compact formatter

### ISSUE THREAD

**Issue by fribeau** [tokio-rs/tracing#1741]
## Bug Report
`compact` formatter only display `timestamp`, `level`and `target`. The log content is not displayed.

### Version
`tracing: 0.1.29`
`tracing-subscriber: 0.3.2`

### Platform
Linux fedora 33

### Crates
`tracing-subscriber`

### Description
```rust
fn main() {
    tracing_subscriber::fmt().compact().init();

    tracing::info!("Message");
}
```

Output only
```bash
2021-11-22T11:04:45.533429Z INFO toto:
```
There is no message

**Comment by Folyd** [comment]
After upgrading the tracing-subscriber from `0.3.1` to `0.3.2`. The same problem occurred.

**Comment by vulpyne** [comment]
This is definitely broken in 0.3.2. 0.3.1 has the expected behavior.

0.3.2 really should be pulled from crates.io et al. In the meantime, the workarounds are either to use a format other than compact or to set the crate version to `=0.3.1` in `Cargo.toml`

**Comment by hawkw** [comment]
Fixed by #1755.
